src/common/InstallResult.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
this file is only related to simulator right? How about to edge folder?

Reply:
OK